.circleci/config.yml

```diff
@@ -108,6 +108,34 @@ jobs:
       MAKE_JOB_COUNT: 8
       WAF: "<< parameters.waf >>"
       NGINX_VERSION: "<< parameters.nginx-version >>"
+  build_arm64_installer:
```

Comment:
> dry run.
> verbose.

👍 

> tests for shell and go.

WDYT of leaving this for later?

> versioning

I guess this depends on [this other conversation](https://github.com/DataDog/nginx-datadog/pull/110#discussion_r1745602587)

> download and verify the go binary.

Can you clarify what you are referring to with this?

> To get data we can get this installer in the wild for tracing and appsec but that requires a why to select the feature needed.

Do you mean letting the customer choose WAF ON/OFF for example? Do you think we should be doing that now given the configuration will be completely correlated to rum injection?

Reply:
> Do you mean letting the customer choose WAF ON/OFF for example? 
Yes, you're absolutely right. WAF requires changes to the configuration, so it might not be the best time to introduce that. However, for APM tracing, only the `load_module <MODULE_PATH>;` is needed. If a customer is using the installer specifically for APM tracing, the current code should be sufficient, right?

Reply:
Should be pretty close, but the config changes as well, we are right now disabling apm with `datadog_disable` and forcing rum on. Are you suggesting adding a parameter determining whether `datadog_disable` & `datadog_rum` should be included or not?